src/main/kotlin/no/telenor/kt/restclient/util/util.kt

```diff
@@ -13,10 +14,16 @@ import java.net.URI
 // --- request body utility---
 
 @Suppress("UNNECESSARY_NOT_NULL_ASSERTION")
-fun RestClient.RequestBodySpec.json(value: Any) = this
+fun RequestBodySpec.json(value: Any) = this
 	.body(value)
 	.contentType(MediaType.APPLICATION_JSON)!!
 
+fun RequestBodySpec.header(header: String, value: Any) : RequestBodySpec = this
+	.header(header, value.toString())
+
+fun RequestBodySpec.headers(vararg headersWithValue: Pair<String, Any>) : RequestBodySpec = this
```

Comment:
nit: the function could also be named just `header`, this way you don't have to remember to use a plural if you want to assign multiple.

Reply:
Well, I like to have the different names given how the Restclient interface is defined originally. Keeps the familiarity intact.